tree/ntuple/src/RFieldUtils.cxx

```diff
@@ -62,7 +62,8 @@ const std::unordered_map<std::string_view, std::string_view> typeTranslationMap{
    {"ULong64_t", "unsigned long long"},
    {"uint64_t", "std::uint64_t"}};
 
-// Natively supported types drop the default template arguments and the CV qualifiers in template arguments.
+// Natively supported types drop the default template arguments and the CV qualifiers in template arguments;
+// the also keep Long64_t as template argument untouched.
 bool IsUserClass(const std::string &typeName)
```

Comment:
I'm not sure I understand this comment for `IsUserClass`? Should the newly added line be "it also keeps" - but then it / the function is not actually modifying anything?!